evennia/utils/utils.py

```diff
@@ -1667,6 +1667,8 @@ def m_len(target):
     # Would create circular import if in module root.
     from evennia.utils.ansi import ANSI_PARSER
     if inherits_from(target, basestring):
+        if "|lt" in target:
+            return len(ANSI_PARSER.strip_mxp(target))
         return len(ANSI_PARSER.strip_mxp(target))
```

Comment:
Why would this help though? With this code, it will use `strip_mxp` if `|lt` is in `target` ... and also if it is not?